r_basics_visualize_data/r_basics_visualize_data.md

```diff
@@ -0,0 +1,864 @@
+
+Of course, there are other visual elements of a data visualization that **wouldn't** automatically change if you change the data you provide. For example, the color of the background of the graph and the label of the axes won't change even if new data is added. R gives us a lot of control over these elements as well. Please note, however: since they aren't "mapped" to a specific quality of the data, we **"set"** them, rather than "map" them, which means we don't put these assignments inside `aes()`.
+
+<div class = "care">
+Don't worry if this distinction seems a bit fuzzy at first -- it can be tricky to grapple with, and will become clearer over time as you gain more experience manipulating visualizations in R. Even advanced coders sometimes set an aesthetic they meant to map, and vice versa.
+</div>
+
+Let's consider an example in a data frame with three columns, called "a", "b", and "c".  
+We can imagine mapping the values in column "a", which are numerical values, to the x axis.  With column "b", also numerical, we can map those values to the y axis.  And for column "c", which has categorical data with "M" and "F" values, we can imagine mapping that to colors.  
+
+We may also want to set the size of the points on the graph to a certain value. While I'm sure you've seen visualizations where the size of a dot corresponds to something about the data, that's not what we want to do here -- we just want to customize the size of the dot to be a little bit bigger than the default, which is why this wouldn't be one of our "mappings" in this particular instance.
+
+
+The mapping in ggplot would be within an **aes()**, or aesthetic mapping, and looks like this:
+
+`aes(x = a, y = b, color = c)`
+
+Note that R automatically figures out reasonable axis limits and a color scale, but you can fine tune this manually.
+
+Here's a visual that might help:
+
+![Small table of data with columns a, b, and c next to a blank x-y coordinate plane. An arrow is drawn from column a to the x-axis, and an arrow from column b to the y-axis. Rows that have a "M" in column C have blue dots, and rows that have an "F" get green dots.](media/abc_aesthetics.png)
+
+
+### Quiz: Aesthetic Mappings
+
+
+Let's do a quick check of your understanding of aesthetic mappings.  As a reminder:
+
+* Data visualization visual elements that **don't change** according to the data are **set** , not mapped.  (for example, the color of the background of the graph, which doesn't vary based on the data).
+* Data visualization elements that **do change** with the data are **mapped** (for example, the height of a bar changes depending on the data, and the x and y position of a point on a scatter plot changes depending on the data).
+
+Here are some aesthetic mappings to consider:
+
+![A handful of aesthetic options including position, shape, size, color, line width, and line type.](media/aesthetic_mappings.png)<!-- style = "max-width:700px" -->

```

Comment:
```suggestion
![6 aesthetic options: position, shape, size, color, line width, and line type.](media/aesthetic_mappings.png)<!-- style = "max-width:700px" -->

```

wording made it unclear to someone w/o visual access whether the alt text actually included all of the mappings shown in the image.